src/core/tokenStore.js

```diff
@@ -116,28 +112,26 @@ export default function tokenStoreFactory(options) {
          *
          * @returns {Promise<Array>}
          */
-        getIndex: function getIndex() {
-            return this.getTokens().then(function(tokens) {
-                return _.chain(tokens)
-                    .values()
-                    .sort(function(t1, t2) {
-                        return t1.receivedAt - t2.receivedAt;
-                    })
-                    .map('value')
-                    .value();
-            });
+        getIndex() {
+            return this.getTokens()
+                .then(tokens => {
+                    return _.chain(tokens)
+                        .values()
+                        .sort((t1, t2) => t1.receivedAt - t2.receivedAt)
+                        .map('value')
+                        .value();
+                });
         },
 
         /**
          * Check whether the given token is in the store
          *
          * @param {String} key - token string
-         * @returns {Boolean}
+         * @returns {Promise<Boolean>}
          */
-        has: function has(key) {
-            return this.getIndex().then(function(latestIndex) {
-                return _.contains(latestIndex, key);
-            });
+        has(key) {
+            return this.getIndex()
+                .then(latestIndex => _.contains(latestIndex, key));
```

Comment:
Can we use ES6 Array.includes() ?
Also, `_.contains` is brittle because it was removed from the lodash API between versions 3 & 4 (we have 2.x)

Reply:
Agree, will fix that